start-aws.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+
+import boto.ec2
+
+
+# AWS region
+aws_region = 'us-west-2'
+
+# Name of security group exposing ports for Buildmaster
+security_group = 'Buildbot staging'
+
+# Key name for EC2 host
+key_name = 'hybrid-master'
+
+
+ec2_image = {
+    'Fedora-x86_64-20-20140407-sda': 'ami-cc8de6fc'
+}
+
+conn = boto.ec2.connect_to_region(aws_region)
+
+# Buildbot master requires 4Gb diskspace.  Set the EBS root disk (disk1)
+# to be 4Gb instead of default (2Gb).  Also add an ephemeral (4Gb) disk
+# (disk2), not currently used.  If we can easily link /var/lib/docker to
+# /mnt/docker, we don't need to increase the size of the EBS disk, which
+# is slightly cheaper (ephemeral storage is free).
+disk1 = boto.ec2.blockdevicemapping.EBSBlockDeviceType()
+disk1.size = 4
+disk2 = boto.ec2.blockdevicemapping.BlockDeviceType(
+    ephemeral_name='ephemeral0')
+diskmap = boto.ec2.blockdevicemapping.BlockDeviceMapping()
```

Comment:
No, I tried it out to prevent having to increase the EBS (at additional cost), but never added the link to make it work. I guess that would have to go in the fabfile? Given the relatively low cost of EBS against the instance running cost, it is probably not worth it. I will remove it.